scripts/purge_queue.py

```diff
@@ -0,0 +1,128 @@
+
+
+def list_queues():
+    sqs = boto3.resource("sqs")
+    client = boto3.client("sqs")
+    all_queues = [q.url for q in sqs.queues.all()]
+    print('\n{:<35s}{:^25s}{:^20s}{:^20s}\n'.format("Queue Name",
+                                                    'Messages Available',
+                                                    'Messages in Flight',
+                                                    'Messages Delayed'))
+    for queue_url in all_queues:
+        _, _, queue_name = urlparse(queue_url).path.rpartition("/")
+        azul_queue_name = config.is_azul_queue(queue_name)
+        if azul_queue_name:
+            queue = client.get_queue_attributes(
+                QueueUrl=queue_url,
+                AttributeNames=['ApproximateNumberOfMessages',
+                                'ApproximateNumberOfMessagesNotVisible',
+                                'ApproximateNumberOfMessagesDelayed'])
+            queue_attributes = queue['Attributes']
+            print('{:<35s}{:^25s}{:^20s}{:^20s}'.format(queue_name,
+                                                        queue_attributes['ApproximateNumberOfMessages'],
+                                                        queue_attributes["ApproximateNumberOfMessagesNotVisible"],
+                                                        queue_attributes["ApproximateNumberOfMessagesDelayed"]))
+
+
+def get_messages_from_queue(options):
+    sqs = boto3.resource("sqs")
+    logger = logging.getLogger(__name__)
+    queue_name = sqs.get_queue_by_name(QueueName=options.queue_name).url
+    output_file_name = options.file_name
+    queue = sqs.Queue(url=queue_name)
+    logger.info("Obtaining messages from queue '%s'", queue_name)
+    logger.info("Writing messages to '%s'", output_file_name)
+
+    visibility_timeout = 60 * 5
+    logger.info("Default visibility timeout '%i'", visibility_timeout)
+    messages = []
+    while True:
+        messages_batch = queue.receive_messages(AttributeNames=['All'],
+                                                MaxNumberOfMessages=10,
+                                                VisibilityTimeout=visibility_timeout)
+        if not messages_batch:  # Nothing left in queue
+            break
+        else:
+            messages.extend(messages_batch)
+    with open(output_file_name, "w") as file:
+        json.dump({"queue": queue_name,
+                   "messages": extract_fields(messages, options.json_body)},
+                  fp=file,
+                  indent=4)
+    logger.info("Wrote %i messages", len(messages))
+    message_batches = list(more_itertools.chunked(messages, 10))
+    if options.delete:
+        logger.info("Removing messages from queue '%s'", queue_name)
+        delete_messages(message_batches, queue)
+    else:
+        logger.info("Return messages to `%s` queue", queue_name)
+        return_messages(message_batches, queue)
+    logging.info("File: {}\nWritten: OK!".format(output_file_name))
```

Comment:
I give up. Why is this using `.format()` while all the other logging code uses percents?

```suggestion
    logging.info("File: {}\nWritten: OK!".format(output_file_name))
```